charts/cmc-citizen-frontend/values.yaml

```diff
@@ -0,0 +1,37 @@
+nodejs:
+  image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest
```

Comment:
is this right image name? nodejs and claim-store!!!

Reply:
no its not!!!! 🙈 will fix